ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -62,7 +62,7 @@
     private static final By LOCATION = By.id("w3");
     private static final By WHO_WHEN_WHERE = By.id("who-when-where");
     private static final By REQUEST_APPOINTMENT = By.linkText("Request Appointment");
-    private static final By APPOINTMENT_EDIT = By.cssSelector("div.info-section:nth-child(4) > div > a:nth-child(3)");
+    private static final By APPOINTMENT_EDIT = By.xpath("/html/body/div[1]/div[3]/div[9]/div[1]/div/div[1]/div[7]/div[1]/a/i");
```

Comment:
please update this to use a more readable element trace than this kind of tree since that can easily break with a slight change on the page